zap/src/main/java/org/zaproxy/zap/utils/ZapTextComponentUndoManager.java

```diff
@@ -49,7 +49,7 @@
  *   <li>{@code UndoManagerPolicy.ALWAYS_DISABLED}
  * </ul>
  *
- * The policy can be changed with the method {@code setUndoManagerPolicy}.
+ * <p>The policy can be changed with the method {@code setUndoManagerPolicy}.
```

Comment:
This could be reverted.

Reply:
Ah yes, that was from the other, bigger PR. IntelliJ wanted a `<p>` but it's pretty random to have it in this changeset